### PR TITLE
Update API

### PR DESCRIPTION
**Pull request by dnasseri**
Fixes #https://github.com/viewthespace/shapeable/issues/2, but doesn't include an app level configuration.
@shaw3257 @CharlesMcMillan 

We still need to update the README

Also, the error message should be more descriptive.

### CODE REVIEW

**Comment by CharlesMcMillan** [comment]
OMG SHAPEABLE

**Comment by dnasseri** [comment]
@shaw3257 I updated the README. Mind taking a look?

**Comment by dnasseri** [comment]
Also changed the version number as we discussed.

File: README.md [hunk 1]
```diff
@@ -51,9 +51,13 @@ app
...
 
-To specify that the versioner should be applied to or excluded from given controller actions, use the :only and :except parameters.
-```
-versioner Serializers::Foo, :except => [:index, :show]
-versioner Serializers::Foo, :only => :delete
-```
 
+The `shape` method returns the serializer constant.
+
+Both `acts_as_shapeable` and `shape` except the following arguments:
```
Comment (shaw3257) [inline README.md]: typo - accept

**Pull request merged**